README.md

```diff
@@ -102,8 +102,8 @@ The `"ServiceHandlerSpan"` is _active_ while it's running FunctionA and Function
 
 **The `ScopeManager` API makes it possible to fetch the `span()` in `FunctionA` and re-activate it in `FunctionB`.** Note that every `Tracer` contains a `ScopeManager`. These are the steps:
 
-1. Start a `Span` via either `startManual` or `startActive(false)` to prevent the `Span` from being finished upon `Scope` deactivation.
-2. In the closure/`Runnable`/`Future`/etc itself, invoke `tracer.scopeManager().activate(span, false)` to re-activate the `Span` and get a new `Scope`, then `deactivate()` it when the `Span` is no longer active (or use try-with-resources for less typing).
+1. Start a `Span` via either `start()` or `startActive(false)` to prevent the `Span` from being finished upon `Scope` deactivation.
```

Comment:
So I think there's some ambiguity here still.  

It seems like "start" and "startActive" are more like "create a new span" and "create a new span using the active span as a parent".  But it doesn't say whether the span is "active" or not when it's started or not.  I don't see know what Scope deactivation is, or how it relates to span activation and deactivation.